bundles/org.openhab.binding.ws980wifi/README.md

```diff
@@ -0,0 +1,92 @@
+# ws980wifi Binding
+
+The ws980wifi binding connects openHAB with ELV WS980WIFI Weather Stations (see de.elv.com).
+It discovers WS980WIFI Weather Stations in the local network and polls the actual data on a regular base.
+The WS980WIFI has to be part of the local network via WiFi integration.
+Only the actual measures are supported, no historical data.
+The binding reads the data of the station, but does not support to write data back (e.g. updates, configuration parameters).
+
+## Supported Things
+
+ELV WS980WIFI Weather Station.
+
+## Discovery
+
+As soon as the WS980WIFI Weather Station is integrated into your local network, this binding is able to discover the weather station automatically and provides  21 measures as channels of the discovered thing. 
+
+## Binding Configuration
+
+The binding provides no configuration parameters.
+
+## Thing Configuration
+
+Discovered things have one non advanced configuration parameter, which can be modified. 
+It is the Refresh Interval. Default is 60 seconds, min. is 30 seconds.
+The advanced configuration parameters should not be modified.
+
+## Channels
+
+| Channel | Type   | Read/Write | Description                 |
+|---------|--------|------------|-----------------------------|
+| tempInside | Number | R | Temperature Inside in °C |
+| tempDewPoint | Number | R | Temperature Outside in °C |
+| tempWindChill | Number | R | Windchill in °C |
+| heatIndex | Number | R | Heat Index in °C |
+| humidityInside | Number | R | Humidity Inside in % |
+| humidityOutside | Number | R | Humidity Outside in % |
+| pressureAbsolut | Number | R | Pressure Absolut in Pascal |
+| pressureRelative | Number | R  | Pressure Relativ in Pascal |
+| windDirection | Number | R | Wind Direction in Grad |
+| windSpeed | Number | R | Wind Speed in km/h |
+| windSpeedGust | Number | R | Wind Speed Gust in km/h |
+| rainLastHour | Number | R | Rain Last Hour in mm |
+| rainLastDay | Number | R | Rain Last Day in mm |
+| rainLastWeek | Number | R | Rain Last Week in mm |
+| rainLastMonth | Number | R | Rain Last Month in mm |
+| rainLastYear | Number | R | Rain Last Year in mm |
+| rainTotal | Number | R | Rain Total in mm |
+| lightLevel | Number | R | Light Level in lux |
+| uvRaw | Number | R | UV Light |
+| uvIndex | Number | R | UV Index |
+
+## Full Example
+
+Thing ws980wifi:ws980wifi:wetterDO `AZ WetterDO`
```

Comment:
Use code block markers:
\```
  Example thing here
\```

Reply:
Have inserted your code block markers at the beginning of the example config, and at the end.